File: Scripts/app/controllers/site_controller.js
var SiteController = (function () {
    var Site = function () {
        this.index = function (sammy) {
        	$.get("/site/main").then(function (data) {
        		$("#body").html(data);
        	});

        };

        this.sign_in = function () {
            $.get("/site/login").then(function (data) {
                $("#body").html(data);
            });
        };

        this.new_acount = function () {
        	$.get("/site/newAcount").then(function (data) {
        		$("#body").html(data);
        	});
        };

        this.prices = function () {
        	$.get("/site/prices").then(function (data) {
        		$("#body").html(data);
        	});
        };
    };
    return Site;
})();

File: Controllers/SiteController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace MenuFacil.Controllers
{
  public class SiteController : Controller
  {
    public ActionResult Index()
    {
        return View();
    }

    public ActionResult Login()
    {
        return View();
    }

    public ActionResult NewAcount()
    {
        return View();
    }

		[HttpPost]
		public ActionResult CreateAcount()
		{
			return View();
		}
				
		public ActionResult Main()
		{
			return View();
		}
		public ActionResult Prices()
		{
			return View();
		}

  }
}
